dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControl/Simplified/SecondaryVoltageControl_INIT.mo

```diff
@@ -0,0 +1,36 @@
+within Dynawo.Electrical.Controls.Voltage.SecondaryVoltageControl.Simplified;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model SecondaryVoltageControl_INIT "Initialisation model for simplified secondary voltage control"
+  import Dynawo.Electrical.SystemBase;
+  import Dynawo.Types;
+
+  parameter Integer NbMaxGen = 50 "Maximum number of generators that should participate in the considered secondary voltage control";
+  parameter Boolean[NbMaxGen] Participate0 = fill(false, NbMaxGen) "If true, the generator participates in the secondary voltage control. If false, the generator does not participate in the secondary voltage control (for each generator connected to the SVC).";
```

Comment:
You can remove the "If false ..." part from the comment : it is simply the opposite of the "If true ..." part.

Reply:
done